packages/components/src/css/component/Table.module.scss

```
@@ -219,6 +166,9 @@
 // A rowspan cell belongs to the first row it covers but stretches across the
 // rest, so it is excluded from the hover highlight (like .tableGroup): otherwise
 // hovering the first row would light up the full-height cell on its own.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)